apps/vmq_webhooks/priv/vmq_webhooks.schema

```diff
@@ -83,3 +83,14 @@
  [{datatype, string},
   hidden,
   {default, "    webhooks    Manage webhooks\n"}]}.
+
+%% @doc Configue the maximum number of HTTP connections maintained in the webhooks pool.
```

Comment:
Should these options be hidden?

Reply:
Configue  -> Configure

Yeah, I'd make them hidden so the vernemq.conf file doesn't get too large, but I'll add the options to the documentation.